<launch>
  <node name="okvis_node" pkg="okvis_ros" type="okvis_node_synchronous" output="screen" args="$(arg config) $(arg bag) $(arg skip)">
    <param name="mesh_file" value="firefly.dae" />
    <param name="camera_topic_0" value="/cam0/image_raw" />
    <param name="imu_topic" value="/imu0" />
  </node>
</launch>
